experimental/ndv4_a100_healthchecks/gen_hpcdiags.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
```

Comment:
I would recommend adding a `set -euo pipefail` here to stop the script any time an error occurs.